recipe/meta.yaml

```diff
@@ -1,13 +1,16 @@
+{% set name = "libffi" %}
 {% set version = "3.2.1" %}
 
 package:
-  name: libffi
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: libffi-3.2.1.tar.gz
-  url: ftp://sourceware.org/pub/libffi/libffi-3.2.1.tar.gz
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: ftp://sourceware.org/pub/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha1: 280c265b789e041c02e5c97815793dfc283fb1e6
+  patches:
+    - configure_includedir_option.patch
 
 build:
   number: 2
```

Comment:
I think you need to bump the build number since the version is staying the same.